docs/dynamic-asg-overview.md

```diff
@@ -113,3 +114,11 @@ Dynamic ASGs is a linux-only feature. Currently there is no vxlan-policy-agent
 equivalent for windows cells. If this is a feature you are interested in, please
 open an issue as a feature request.
 
+### 8. Why aren't my security-group updates taking a very long time to be detected and synced through to policy-server?
```

Comment:
Is this supposed to be "aren't"?  It seems like it should be "why are ... taking a very long time".

Reply:
uh huh.